### ISSUE TITLE
Add example for `config.ld` to LDoc's documentation

### ISSUE THREAD

**Issue by NicolBolas** [stevedonovan/LDoc#269]
LDoc's documentation is pretty decent overall. However, there is one problem: at no point do you show or explain what a `config.ld` file looks like. Perhaps this was a concept created by LuaDoc, but as someone who's never actually used that program, having

It might also be a good idea to add a way to generate a basic `config.ld` file. Much like Doxygen's -g option, it would generate a file of defaults, with a comment explaining what each field means.

**Comment by gotonickpappas** [comment]
Yeah, there is a need for an example. In any case, you can get a totally bare-bones example there(see link below) to get you started.
https://github.com/stevedonovan/LDoc/blob/master/tests/example/config.ld
